ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -32384,14 +32384,35 @@ periodics:
       - --lease-server-credentials-file=/etc/boskos/credentials
       - --report-credentials-file=/etc/report/credentials
       - --secret-dir=/usr/local/pull-secret
-      - --target=release-payload-install-analysis
+      - --target=release-payload-install-analysis-4.11
       command:
       - ci-operator
       env:
       - name: GOOGLE_SA_CREDENTIAL_FILE
         value: /var/run/secrets/google-serviceaccount-credentials.json
       - name: JOB_START_TIME
       - name: PAYLOAD_TAG
+      - name: UNRESOLVED_CONFIG
+        value: |
+          resources:
+            '*':
+              limits:
+                memory: 6Gi
+              requests:
+                cpu: 100m
+                memory: 200Mi
+          tests:
+          - as: release-payload-install-analysis-4.11
+            steps:
+              env:
+                GOOGLE_SA_CREDENTIAL_FILE: "$(GOOGLE_SA_CREDENTIAL_FILE)"
+                JOB_START_TIME: "$(JOB_START_TIME)"
+                PAYLOAD_TAG: "$(PAYLOAD_TAG)"
+                WORKING_DIR: "$(ARTIFACT_DIR)/release-analysis-install-analysis"
+                MINIMUM_SUCCESSFUL_COUNT: "$(MINIMUM_SUCCESSFUL_COUNT)"
+                TEST_GROUP: "$(TEST_GROUP)"
```

Comment:
No need for MINIMUM_SUCCESSFUL_COUNT and TEST_GROUP

Reply:
done